CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 ## CHANGELOG
 
-### 2.0.0 (in progress)
+### 2.0.0 (2020-08-07)
 **SDK**
+- **Feature:** Mini App is now downloaded as a ZIP archive and extracted. This should improve the initial launch time on a Mini App with many files.
+- **Feature:** Added custom dialog's for `window.alert`, `window.prompt`, and `window.confirm`. These look and behave the same as the default WebView dialogs with the exception that the dialog title is now set to the Mini App's name.
```

Comment:
`dialog's` -> `dialog`? 
Please ignore it if I am wrong.

Reply:
Thanks, I will update this.